boards.txt

```diff
@@ -50155,3 +50155,157 @@ cyobot_v2_esp32s3.menu.ZigbeeMode.zczr.build.zigbee_mode=-DZIGBEE_MODE_ZCZR
 cyobot_v2_esp32s3.menu.ZigbeeMode.zczr.build.zigbee_libs=-lesp_zb_api.zczr -lzboss_stack.zczr -lzboss_port.remote
 
 ##############################################################
+
+# Satellite 1 CORE (rev 5.1) 
+
+sat1core_rev5_1.name=Satellite 1 CORE (rev 5.1) 
+
+sat1core_rev5_1.bootloader.tool=esptool_py
+sat1core_rev5_1.bootloader.tool.default=esptool_py
+
+sat1core_rev5_1.upload.tool=esptool_py
+sat1core_rev5_1.upload.tool.default=esptool_py
+sat1core_rev5_1.upload.tool.network=esp_ota
+
+sat1core_rev5_1.upload.maximum_size=1310720
+sat1core_rev5_1.upload.maximum_data_size=327680
+sat1core_rev5_1.upload.flags=
+sat1core_rev5_1.upload.extra_flags=
+sat1core_rev5_1.upload.use_1200bps_touch=false
+sat1core_rev5_1.upload.wait_for_upload_port=false
+
+sat1core_rev5_1.serial.disableDTR=false
+sat1core_rev5_1.serial.disableRTS=false
+
+sat1core_rev5_1.build.tarch=xtensa
+sat1core_rev5_1.build.bootloader_addr=0x0
+sat1core_rev5_1.build.target=esp32s3
+sat1core_rev5_1.build.mcu=esp32s3
+sat1core_rev5_1.build.core=esp32
+sat1core_rev5_1.build.variant=satellite1_core_rev5_1
+sat1core_rev5_1.build.board=SATELLITE1_CORE_REV5_1
+
+sat1core_rev5_1.build.usb_mode=1
+sat1core_rev5_1.build.cdc_on_boot=1
+sat1core_rev5_1.build.msc_on_boot=0
+sat1core_rev5_1.build.dfu_on_boot=0
+sat1core_rev5_1.build.f_cpu=240000000L
+sat1core_rev5_1.build.flash_size=16MB (128Mb)
+sat1core_rev5_1.build.flash_freq=80m
+sat1core_rev5_1.build.flash_mode=dio
+sat1core_rev5_1.build.boot=qio
+sat1core_rev5_1.build.boot_freq=80m
+sat1core_rev5_1.build.partitions=default
+sat1core_rev5_1.build.defines=-DBOARD_HAS_PSRAM
+sat1core_rev5_1.build.loop_core=
+sat1core_rev5_1.build.event_core=
+sat1core_rev5_1.build.psram_type=opi
+sat1core_rev5_1.build.memory_type={build.boot}_{build.psram_type}
+
+sat1core_rev5_1.menu.PSRAM.opi=Enabled
+sat1core_rev5_1.menu.PSRAM.opi.build.defines=-DBOARD_HAS_PSRAM
+sat1core_rev5_1.menu.PSRAM.opi.build.psram_type=opi
+
+sat1core_rev5_1.menu.FlashMode.qio=QIO 80MHz
+sat1core_rev5_1.menu.FlashMode.qio.build.flash_mode=dio
+sat1core_rev5_1.menu.FlashMode.qio.build.boot=qio
+sat1core_rev5_1.menu.FlashMode.qio.build.boot_freq=80m
+sat1core_rev5_1.menu.FlashMode.qio.build.flash_freq=80m
```

Comment:
I think you can run a OPI Flash mode as your module has OPI PSRAM, if you don't have something custom. Can you check?